### ISSUE TITLE
Is there a plan to replace  ZKWeb.System.Drawing with System.Drawing.Common

### ISSUE THREAD

**Issue by qqbuby** [dotnetcore/NPOI#50]

**Comment by yuleyule66** [comment]
The current API of `System.Drawing.Common` is incomplete, we need to wait until .net core 2.1 releases.

**Comment by qqbuby** [comment]
any updates?

**Comment by tapjyotmakkar** [comment]
Not sure if my problem is relevant to this thread but on generating a .xlsx from a docker web container running on linux am getting the following error: 
**Exception thrown: System.TypeInitializationException' in ZKWeb.System.Drawing.dll**

Any ideas/suggestions ??

**Comment by qqbuby** [comment]
@tapjyotmakkar 

Maybe you forget to setup `libgdplus` as below:

```dockerfile

RUN apt-get update \
    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
        libgdiplus \
    && rm -rf /var/lib/apt/lists/* \
    && ln -sf /usr/lib/libgdiplus.so /usr/lib/gdiplus.dll
```

**Comment by yuleyule66** [comment]
Fixed in v1.2.0
